pkg/route/api/validation/validation.go

```diff
@@ -32,6 +32,11 @@ func ValidateRoute(route *routeapi.Route) fielderrors.ValidationErrorList {
 		result = append(result, fielderrors.NewFieldInvalid("path", route.Spec.Path, "path must begin with /"))
 	}
 
+	if len(route.Spec.Path) > 0 && route.Spec.TLS != nil &&
+		route.Spec.TLS.Termination == routeapi.TLSTerminationPassthrough {
```

Comment:
We have to be careful with tightening validation on things "that would never have worked"… we can do it as long as we're sure that no automated systems will be hot looping or logging thousands of events/validation failures if a previously "valid" persisted object suddenly doesn't pass validation after an upgrade